scripts/new_bare_repo

```diff
@@ -43,7 +43,7 @@ elif [[ ${CWD} != *${REPO}* ]]; then
 else
     git init
     cp -u /cds/sw/tools/bin/eco_tools/gitignore.template .gitignore
-    git add *.cfg Makefile .gitignore
+    git add -- *.cfg Makefile .gitignore
```

Comment:
SC2035 (info): Use ./*glob* or -- *glob* so names with dashes won't become options.
Realistically, I don't think this would ever cause an issue here so we could add it to .shellcheckrc